### ISSUE TITLE
A minor error in the final plot of the section 16.2

### ISSUE THREAD

**Issue by OmidGhasemi21** [ASKurz/Doing-Bayesian-Data-Analysis-in-brms-and-the-tidyverse#37]
Hi @ASKurz,

I think there is a minor problem with the final plot of the t-distribution section (16.2, or Figure 16.6 of the book). You can find the plot from the bookdown below. As you can see, the left column shows the percentage of the distribution under σ = ±1. The right column shows the area under the curve that is needed to include 68.27% of the distribution. Consider the first row, when ν = inf (in which t approaches a normal distribution) 68.27% of the distribution is under σ = ±1, but this is not the case for the right column. Moreover, the grey areas for plots in the right column do not match those in the book (the plots in the left column are fine). 

![unnamed-chunk-35-1](https://user-images.githubusercontent.com/45413424/127247548-95b0dbe5-2806-4845-a692-61d417552926.png)

This problem can simply be solved by changing the order of values when defining the `ymin` variable when you create the primary data for the right column plots. I commented out the problematic line and added a new line there.

```r
# the primary data
d <-
  crossing(y  = seq(from = -8, to = 8, length.out = 1e3),
           nu = c(Inf, 5, 2, 1)) %>%
  mutate(label = str_c("nu == ", nu) %>% 
           factor(., levels = c("nu == Inf", "nu == 5", "nu == 2", "nu == 1")))

# the subplot
p1 <-
  d %>% 
  ggplot(aes(x = y)) +
  geom_area(aes(y = dt(y, df = nu)),
            fill = bp[2]) +
  geom_area(data = . %>% filter(y >= -1 & y <= 1),
            aes(y = dt(y, df = nu)),
            fill = bp[1]) +
  # note how this function has its own data
  geom_text(data = tibble(
    y     = 0,
    text  = c("68%", "64%", "58%", "50%"),
    label = factor(c("nu == Inf", "nu == 5", "nu == 2", "nu == 1"))),
    aes(y = .175, label = text),
    color = "white") +
  scale_y_continuous(expression(p(y)), expand = expansion(mult = c(0, 0.05)), breaks = c(0, .2, .4)) +
  labs(subtitle = "Shaded from y = - 1 to y = 1") +
  coord_cartesian(xlim = c(-6, 6)) +
  facet_wrap(~ label, ncol = 1, labeller = label_parsed)


# Right Column
# the primary data
d <-
  tibble(nu   = c(Inf, 5, 2, 1),
         #ymin = c(-1.84, -1.32, -1.11, -1) # the order of values here is not correct
         ymin = c(-1, -1.11, -1.32, -1.84)) %>%
  mutate(ymax = -ymin) %>% 
  expand(nesting(nu, ymin, ymax),
         y = seq(from = -8, to = 8, length.out = 1e3)) %>%
  mutate(label = factor(str_c("nu==", nu), 
                        levels = str_c("nu==", c(Inf, 5, 2, 1))))

# the subplot
p2 <-
  d %>% 
  ggplot(aes(x = y)) +
  geom_area(aes(y = dt(y, df = nu)),
            fill = bp[2]) +
  geom_area(data = . %>% 
              # notice our `filter()` argument has changed
              filter(y >= ymin & y <= ymax),
            aes(y = dt(y, df = nu)),
            fill = bp[1]) +
  annotate(geom = "text", 
           x = 0, y = .175, 
           label = "68%", color = "white") +
  scale_y_continuous(expression(p(y)), expand = expansion(mult = c(0, 0.05)), breaks = c(0, .2, .4)) +
  labs(subtitle = "Shaded for the middle 68.27%") +
  coord_cartesian(xlim = c(-6, 6)) +
  facet_wrap(~ label, ncol = 1, labeller = label_parsed)

p1 + p2
```

This code will make the plot that matches the one in the book:

![tplot](https://user-images.githubusercontent.com/45413424/127247708-5c492bff-2dd3-4386-ba9c-d96716669e3d.png)

Hope that helps.
Omid

**Comment by ASKurz** [comment]
Ooo, this looks like a good catch; thanks @OmidGhasemi21!

**Comment by ASKurz** [comment]
Yep, you're right. You can find a draft of the corrected version of Section 16.2 [here](https://github.com/ASKurz/Doing-Bayesian-Data-Analysis-in-brms-and-the-tidyverse/blob/master/in_the_works/16.2.md).

**Comment by ASKurz** [comment]
This has been integrated into the primary `.Rmd` file with [this commit](https://github.com/ASKurz/Doing-Bayesian-Data-Analysis-in-brms-and-the-tidyverse/commit/d7f5e73a0768ca83e9755a4af78d76daad72dc95).
